demo/rendezvous.env

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Intel Corporation
```

Comment:
2019 => 2020 (considering it is a new file)

Reply:
Done